examples/local-access/setup-external-connection.ipynb

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
<p>I think the settings here should be changed to fit into freemium?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/121/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I thought these settings would be allowed but let me test and see- you're probably right!